kitsune/wiki/tests/test_parser.py

```diff
@@ -192,11 +192,11 @@ def test_internal_links(self):
         # Both internal links should link to the same article
         self.assertEqual(
             p.parse("[[%s]]" % doc.title),
-            '<p><a href="/en-US/kb/%s">%s</a>\n</p>' % (doc.slug, doc.title),
```

Comment:
why does this need changing?

Reply:
Because
`'<p><a href="/en-US/kb/real-article/">Real article</a>\n</p>' != '<p><a href="/en-US/kb/real-article">Real article</a>\n</p>'
- <p><a href="/en-US/kb/real-article/">Real article</a>`